content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,204 @@
+- :guilabel:`Advanced price rules (discounts, formulas)`.
+
+The **first** option allows you to set different prices per customer *segment*, i.e. registered
+customers, gold customers, regular customers, etc. The **second** option allows you to set *price
+change* rules such as **discounts**, **margins**, **roundings**, etc.
+
+Foreign currency
+~~~~~~~~~~~~~~~~
+
+If you are selling in **multiple currencies** and have pricelists in foreign currencies, customers
+can select their coresponding pricelist anywhere on the :guilabel:`Shop` page from the drop-down
+menu next to the **search bar**.
+
+.. image:: price_management/price-pricelists.png
+   :align: center
+   :alt: Pricelists selection
+
+.. seealso::
+   - :doc:`../../../sales/sales/products_prices/prices/pricing`
+   - :doc:`../../../sales/sales/products_prices/prices/currencies`
+
+Discount and loyalty programs
+=============================
+
+It is possible to create promotion and loyalty programs, which offer different options from
+pricelists. To do so, go to :menuselection:`Website app --> eCommerce --> Discount & Loyalty`, and
+:guilabel:`Create` a program from scratch or by using the templates. You can also modify an existing
+program by clicking on it.
+
+.. image:: price_management/price-discount-loyalty.png
+   :align: center
+   :alt: Discount and loyal program templates
+
+.. _pricing_management/program-types:
+
+The different program types are:
+
+- :guilabel:`Coupons`: Generate and share single-use, coupon codes that grant access to rewards;
+- :guilabel:`Next Order Coupons`: Generate and share single-use, coupons that grant access to
+  rewards on the next order;
+- :guilabel:`Promotions`: Set conditional rules when ordering, which grant access to reward for the
+  customers;
+- :guilabel:`Discount Code`: Set codes which when entered upon checkout grants customers a discount;
+- :guilabel:`Loyalty Cards`: Customers receive points when buying, which they can exchange for
+  rewards on their current or future order;
+- :guilabel:`Buy X Get Y`: Grant 1 credit for each item bought, then reward customers with Y item in
+  exchange of X credits.
+
+Configuration
+-------------
+
+When creating or modifying a program, the options available depend on the :guilabel:`Program Type`
+selected. Therefore, some programs may have additional options related to their type.
+
+- :guilabel:`Program Name`: Enter the name of the program, this is not visible by the customer;
+- :guilabel:`Program Type`: The :ref:`program type <pricing_management/program-types>` you want to
+  use;
+- :guilabel:`Currency`: The currency you wish to use for that program;
+- :guilabel:`Points Unit`: The unit you wish to use for the :guilabel:`Loyalty Cards`. You are free
+  to enter any name you want;
+- :guilabel:`Validity`: The date until which the program is valid;
+- :guilabel:`Limit Usage`: The limited number of times the program can be used during the
+  :guilabel:`Validity`;
```

Comment:
```suggestion
- :guilabel:`Loyalty Cards`: Customers receive points when buying, which they can exchange for
  rewards on their current or future order;
- :guilabel:`Buy X Get Y`: Grant 1 credit for each item bought, then reward customers with Y item in
exchange of X credits.

Configuration
-------------

When creating or modifying a program, the options available depend on the :guilabel:`Program Type`
selected. Therefore, some programs may have additional options related to their type.

- :guilabel:`Program Name`: Enter the name of the program, this is not visible by the customer;
- :guilabel:`Program Type`: The :ref:`program type <pricing_management/program-types>` you want to
  use;
- :guilabel:`Currency`: The currency you wish to use for that program;
- :guilabel:`Points Unit`: The unit you wish to use for the :guilabel:`Loyalty Cards`. You are free
  to enter any name you want;
- :guilabel:`Validity`: The date until which the program is valid;
- :guilabel:`Limit Usage`: The limited number of times the program can be used during the
  :guilabel:`Validity`;
```

Reply:
I'm not sure what is the problem here. I see on line 146 you deleted a blank space, but on my end the spacing is just as it should be (and so is the character limit)
![jaae](https://user-images.githubusercontent.com/104567387/205002046-cc994465-636e-4c5d-b239-c9703f501b61.png)